Adding a sample daemon calls web API using MSI (#191)
Azure-Samples/active-directory-dotnetcore-daemon-v2: 6-Call-OwnApi - ManagedIdentity/AppCreationScripts/AppCreationScripts.md

### Usage pattern for tests and DevOps scenarios

The `Configure.ps1` will stop if it tries to create a Microsoft Entra application which already exists in the tenant. For this, if you are using the script to try/test the sample, or in DevOps scenarios, you might want to run `Cleanup.ps1` just before `Configure.ps1`. This is what is shown in the steps below.

## How to use the app creation scripts?

### Pre-requisites

1. Powershell 7 or later
    1. You can follow the instrunctions to install PowerShell at [this link](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.3)
1. Open PowerShell (On Windows, press  `Windows-R` and type `PowerShell` in the search window)

### (Optionally) install Microsoft.Graph.Applications PowerShell modules

The scripts install the required PowerShell module (Microsoft.Graph.Applications) for the current user if needed. However, if you want to install if for all users on the machine, you can follow the following steps:

> 1. If you have never done it already, in the PowerShell window, install the Microsoft.Graph.Applications PowerShell modules. For this:

Reviewer: ```suggestion
1. If not already installed, in the PowerShell window, install the Microsoft.Graph.Applications PowerShell modules. For this:
```

    cd AppCreationScripts
    ```

1. Run the scripts. See below for the [four options](#four-ways-to-run-the-script) to do that.
1. Open the Visual Studio solution, and in the solution's context menu, choose **Set Startup Projects**.
1. select **Start** for the projects

You're done!

### Two ways to run the script

We advise four ways of running the script:

- Interactive: you will be prompted for credentials, and the scripts decide in which tenant to create the objects,
- Interactive in specific tenant: you will provide the tenant in which you want to create the objects and then you will be prompted for credentials, and the scripts will create the objects,

Here are the details on how to do this.

#### Option 1 (interactive)

- Just run ``.\Configure.ps1``, and you will be prompted to sign-in (email address, password, and if needed MFA).
- The script will be run as the signed-in user and will use the tenant in which the user is defined.

Note that the script will choose the tenant in which to create the applications, based on the user. Also to run the `Cleanup.ps1` script, you will need to re-sign-in.

#### Option 2 (Interactive, but create apps in a specified tenant)

>   if you want to create the apps in a particular tenant, you can use the following option:

Reviewer 2: super nit: remove leading space and capitalize.
